admin/sql/create_tables.sql

```diff
@@ -42,4 +42,24 @@ CREATE TABLE "user" (
 );
 ALTER TABLE "user" ADD CONSTRAINT user_musicbrainz_id_key UNIQUE (musicbrainz_id);
 
+CREATE TABLE dataset (
+  id          UUID,
+  name        VARCHAR NOT NULL,
+  description TEXT,
+  author      INT, -- FK to user
+  created     TIMESTAMP WITH TIME ZONE DEFAULT NOW()
```

Comment:
We need to make sure we have a future task to be able to "publish" datasets. They should be private by default